### PR TITLE
Development- Building a Old Version Apple Page Copy

### PR DESCRIPTION
**Pull request by MarilenaRoque**
On this branch, I included all the HTML structure and CSS style to build a copy of an old version of the Apple page, applying all the theory content learned about the use of Image Background and Gradients. Including: 

#Main Section
Image of background
Position and Responsiveness equal to the original

-Navbar
Used gradient to its background
Used just one image for all the Icons in the Navbar using the background property
Set up the color change when the Items are hover. 
Included the search input
Set up the change of size and background color for the Search Input
Included all the links for the original page for the Navbar items

-Titles
Included and positioned images to represent the titles like in the original page
Set up the underline effect when the link is hover.

#Promo Section
Included and positioned the divisions and their background
Added responsiveness like in the original site

#Footer
Included and positioned all the divisions and their elements
Styled and set up all the links and their effect when hover them
Included Icon

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+              <!-- Promo Section -->
+
+      <section class="promo content-display">
+          <div class="promo-item iphoneRose"></div>
+          <div class="promo-item iphoneBlack"></div>
+          <div class="promo-item mac30"></div>
+          <div class="promo-item responsibility"></div>
+      </section>
+
```
Comment (sumancrest0001) [inline index.html]: **STATUS: CHANGES REQUIRED
Impressive work @MarilenaRoque  :clap:  :clap: :+1:** 
You have done a terrific job and ticked all the requirements. Everything looks perfect but there is one minor issue. We have to fix one HTML validator error.
Please have a look :point_down: 
Screenshot from 2020-05-13 08-12-55
[TIPS] We have to add atleast one `<h>` tag inside a `<section>` tag.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
Kindly make the requested change and submit the project for another review. 
Need Help? DM me on slack (**Suman Shrestha**)

Happy coding !!!

**Comment by MarilenaRoque** [comment]
@sumancrest0001 Thank you so much for your review.
 
The validator warning was fixed!

**Review by sumancrest0001 — APPROVED** [review]
**Status: Project Approved
Well Done @MarilenaRoque** :clap:  :clap: 

It's a green signal from my side. Go ahead and merge the branch to master. All the best with the next project.
mission

**Happy coding !**

**Pull request merged**